custom_components/myskoda/sensor.py

```diff
@@ -274,6 +274,10 @@ def icon(self):  # noqa: D102
             return "mdi:power-plug-battery"
         return "mdi:power-plug"
 
+    @property
+    def available(self):  # noqa: D102
+        return self._status().state is not None
```

Comment:
Can it actually ever be `None`? Since `ChargingStatus._status()` will raise an exception if the status is `None`